app/js/lib/intent-parser/refiners/query-refiner.js

```diff
@@ -0,0 +1,15 @@
+export default class QueryRefiner {
+  refine(obj = {}) {
+    if (obj.cleaned.match(/^(?:What|Where)/i)
```

Comment:
Would be cool if this could support 'when'.

'When should I pick up the VIP?'

Maybe for the future though.